app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt

```diff
@@ -54,6 +54,14 @@ class TopUpWalletPresenter(
     }
 
     override fun onBankTransferClicked() {
+        if (!strigaSignupFeatureToggle.isFeatureEnabled) {
+            launch {
+                val tokenToBuy = userInteractor.getSingleTokenForBuy()
+                tokenToBuy?.let {
+                    view?.navigateToBuyWithTransfer(it)
+                }
+            }
+        }
```

Comment:
I think there is missing `return` statement

Reply:
Shouldn't we hide the transfer item if striga is not enabled?

Reply:
no
in new logic we should not

Reply:
@gslevinkov hah, yeah, it hides the item in attachView if striga is disabled and user is web3-authorized
@kabramovp2p have you checked this?